lib/gitarro/backend.rb

```diff
@@ -192,6 +192,15 @@ def open_newer_prs
     prs
   end
 
+  # public forcing to run the test
```

Comment:
Embeding the force test to pr option is imho better because now if you run the force tests, without the pr number option, it, will always testing the 1st pr which doesn't make any value

Reply:
Good catch. I added the missing condition.
`return unless @force_test and @pr_number.any?`